features/step_definitions/back_office/upper_tier/assisted_digital_new_upper_tier_registration_steps.rb

```diff
@@ -100,51 +100,6 @@
   @back_app.check_details_page.submit
 end
 
-Given(/^a limited company with companies house number "([^"]*)" is registered as an upper tier waste carrier$/) do |ch_no|
```

Comment:
This was removed because was used only by the test I refactored so you don't need this any more :) There are others that still go through the old app though.

Reply:
I mean, I can avoid deleting this, but once this PR is merged there won't be any scenario using it any more

Reply:
Ah OK - thanks for confirming! :D

Reply:
Also, given that I have re-defined it to go through the new app (https://github.com/DEFRA/waste-carriers-acceptance-tests/pull/269/files#diff-52cf972af9ba08976f254c0998e41497R20) I can't keep this. Cucumber will complain about 2 steps that are named the same